dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/ProcessDefinitionService.java

```diff
@@ -427,4 +427,9 @@ ProcessDefinition updateSingleProcessDefinition(User loginUser,
      * @return variables data
      */
     Map<String, Object> viewVariables(User loginUser, long projectCode, long code);
+
+    void saveProcessLineage(List<TaskDefinitionLog> taskDefinitionLogList,
+                            long projectCode,
+                            long processDefinitionCode,
+                            int processDefinitionVersion);
```

Comment:
Keep the parameter range from large to small
```suggestion
    void saveProcessLineage(long projectCode,
                            long processDefinitionCode,  int processDefinitionVersion, 
                            List<TaskDefinitionLog> taskDefinitionLogList);
```

Reply:
Should change to `ProcessTaskLineage`? Because it not just keep the workflow relations.

Reply:
Good suggestion. Will fix it.